addons/test_mail/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     'description': """This module contains tests related to mail. Those are
 contained in a separate module as it contains models used only to perform
 tests independently to functional aspects of other models. """,
-    'depends': ['test_performance', 'mail', 'mail_bot'],
+    'depends': ['test_performance', 'mail', 'mail_bot_hr'],
```

Comment:
Argl. I prefer to lessen dependencies of test_mail and keep only mail_bot . HR should not impact mail-related test modules.